.github/workflows/swift-toolchain.yml

```diff
@@ -199,8 +212,19 @@ jobs:
             fi
           fi
 
-          echo windows_build_runner=${{ vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
-          echo compilers_build_runner=${{ vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          WINDOWS_BUILD_RUNNER=""
+          COMPILERS_BUILD_RUNNER=""
+
+          if [[ ${{ vars.USE_CIRUN }} == "true" || ${{ vars.USE_CIRUN }} == true ]]; then
+            WINDOWS_BUILD_RUNNER="cirun-win11-23h2-pro-x64-16-2024-05-17"
+            COMPILERS_BUILD_RUNNER="cirun-win11-23h2-pro-x64-64-2024-05-17"
+          else
+            WINDOWS_BUILD_RUNNER="${{ vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }}"
```

Comment:
Does this not already allow us to use CIRUN by changing the github vars? Is that enough (it would allow us to not diverge from compnerd/swift-build)

Reply:
Otherwise, it would be great if we could make the divergence more in declarative code than imperative, or anything that would make merging easier. I'm not sure what's possible though.

Reply:
Yeah that's a good point. I don't think we need this.